config/settings/base.py

```diff
@@ -5,8 +5,11 @@
 import os
 from os.path import join
 
+import djcelery
```

Comment:
`djcelery` is very much deprecated. I think we should rather try to do without it, unless we really need one of the features.